src/dxvk/rtx_render/rtx_materials.h

```diff
@@ -1478,6 +1486,7 @@ struct LegacyMaterialData {
   uint32_t colorTextureSlot[kMaxSupportedTextures] = { kInvalidResourceSlot };
 
   XXH64_hash_t m_cachedHash = kEmptyHash;
+  bool m_overrideHash = false;
```

Comment:
small nitpick - this name sounds like the variable contains the override hash.  `m_isHashOverridden` and `IsHashOverridden()` would make it more clear you're talking about a boolean, not the actual hash value.

Reply:
Indeed! Changed that and squashed all the commits 👍